### ISSUE TITLE
MR #3817 Breaks launching scrcpy-console using the .desktop file

### ISSUE THREAD

**Issue by gardotd426** [Genymobile/scrcpy#4367]
- [x ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Arch Linux
 - scrcpy version: Master
 - installation method: From source via AUR
 -  model: N/A
 - Android version: N/A

**Describe the bug**
https://github.com/Genymobile/scrcpy/pull/3817 breaks `scrcpy-console`, because there are one too many backslashes in the `Exec` line. 

`/bin/bash --norc --noprofile -i -c "\"\\$SHELL\" -i -c scrcpy || read -p 'Press Enter to quit...'"` 

errors out, because it reads it as `\/bin/zsh` (or `\/bin/bash`, `\/bin/fish`, etc) and gives you a `\/bin/zsh: no such file or directory` error. 

Removing one of the backslashes before `$SHELL` fixes the problem completely and you can launch scrcpy-console from the .desktop file as intended.  

```
bash: \/bin/zsh: No such file or directory
Press Enter to quit...
```
Here's what happens when I run `/bin/bash --norc --noprofile -i -c "\"\$SHELL\" -i -c scrcpy || read -p 'Press Enter to quit...'"`:

```
scrcpy 2.1.1 <https://github.com/Genymobile/scrcpy>
ERROR: Could not find any ADB device
ERROR: Server connection failed
Press Enter to quit...
```
Obviously I don't have adb running so that error can be ignored, when I have adb running and a device connected, it functions. 

This may be a new issue due to Plasma updating or , but during my update I got this warning and many similar other warnings about other desktop files:

```
kf.config.core: "KConfigIni: In file /usr/share/applications/scrcpy-console.desktop, line 8: " "Invalid escape sequence \"\\\"\"."

kf.config.core: "KConfigIni: In file /usr/share/applications/scrcpy-console.desktop, line 8: " "Invalid escape sequence \"\\\"\"."
```

**Comment by rom1v** [comment]
Thank you for your report.

After #4130 (on `dev` branch), the .desktop scripts have been simplified a bit.

Could you please open a PR based on `dev` branch which fixes this issue?

**Comment by gardotd426** [comment]
I sure can. Its 3 AM here so I'll do it when I wake up.

**Comment by rom1v** [comment]
Did you have time to take a look?
